Cargo.toml

```
@@ -36,6 +36,8 @@ qlog = { version = "0.13", default-features = false }
 quinn-udp = { version = "0.5.6", default-features = false, features = ["direct-log", "fast-apple-datapath"] }
 regex = { version = "1.9", default-features = false, features = ["unicode-perl"] }
 static_assertions = { version = "1.1", default-features = false }
+strum = { version = "0.26", default-features = false }
+strum_macros = { version = "0.26", default-features = false }
```

https://github.com/Peternator7/strum?tab=readme-ov-file#including-strum-in-your-project

```suggestion
strum = { version = "0.26",  default-features = false, features = ["derive"] }
```

Should work as well. One less version to update. Not a strong opinion.